contrib/dokku-installer.rb

```diff
@@ -3,21 +3,26 @@
 require "open3"
 
 if ARGV[0] == "onboot"
-  File.open("/etc/init/dokku-installer.conf", "w") do |f|
-    f.puts "start on runlevel [2345]"
-    f.puts "exec #{File.absolute_path(__FILE__)} selfdestruct"
-  end if File.directory?("/etc/init")
-  File.open("/etc/systemd/system/dokku-installer.service", "w") do |f|
-    f.puts "[Unit]"
-    f.puts "Description=Dokku web-installer"
-    f.puts ""
-    f.puts "[Service]"
-    f.puts "ExecStart=#{File.absolute_path(__FILE__)} selfdestruct"
-    f.puts ""
-    f.puts "[Install]"
-    f.puts "WantedBy=multi-user.target"
-    f.puts "WantedBy=graphical.target"
-  end if File.directory?("/etc/systemd/system")
+  if File.realpath("/sbin/init") != "/sbin/init" then
```

Comment:
Yes, for this time. I will correct it and re-work in future. I am planning
create a RHEL port, so I will make more changes in this way. Thanks.

2015-12-23 8:25 GMT+01:00 Jose Diaz-Gonzalez <notifications@github.com>:

> In contrib/dokku-installer.rb
> <https://github.com/dokku/dokku/pull/1636#discussion_r48325969>:
>
> > -  File.open("/etc/init/dokku-installer.conf", "w") do |f|
> > -    f.puts "start on runlevel [2345]"
> > -    f.puts "exec #{File.absolute_path(__FILE__)} selfdestruct"
> > -  end if File.directory?("/etc/init")
> > -  File.open("/etc/systemd/system/dokku-installer.service", "w") do |f|
> > -    f.puts "[Unit]"
> > -    f.puts "Description=Dokku web-installer"
> > -    f.puts ""
> > -    f.puts "[Service]"
> > -    f.puts "ExecStart=#{File.absolute_path(__FILE__)} selfdestruct"
> > -    f.puts ""
> > -    f.puts "[Install]"
> > -    f.puts "WantedBy=multi-user.target"
> > -    f.puts "WantedBy=graphical.target"
> > -  end if File.directory?("/etc/systemd/system")
> > +  if File.realpath("/sbin/init") != "/sbin/init" then
>
> Are you no longer interested in making this change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dokku/dokku/pull/1636/files#r48325969>.
>



-- 
Jindrich Skupa
Tel: +420602807773
E-mail: jindrich.skupa@gmail.com